packages/belos/tpetra/test/MINRES/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Tpetra_minres_diag
+  SOURCES test_minres_diag.cpp
+  COMM serial mpi
+  STANDARD_PASS_OUTPUT
+)
+
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Tpetra_minres_indefinite
+  SOURCES test_minres_indefinite.cpp
+  COMM serial mpi
+  ARGS
+    "--debug --verbose --tol=1e-5"
+  STANDARD_PASS_OUTPUT 
+)
+
+ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_Triutils)
+IF (${PACKAGE_NAME}_ENABLE_Triutils)
+  TRIBITS_ADD_EXECUTABLE_AND_TEST(
+    Tpetra_minres_hb
+    SOURCES test_minres_hb.cpp 
+    COMM serial mpi
+    ARGS
+      "--verbose --filename=bcsstk14.hb --tol=1e-5"
+      "--verbose --filename=bcsstk14.hb --num-rhs=2 --tol=1e-5"
+    STANDARD_PASS_OUTPUT 
+  )
+
+  ASSERT_DEFINED(Anasazi_SOURCE_DIR)
+  TRIBITS_COPY_FILES_TO_BINARY_DIR(CopyTestMinresFiles
+    SOURCE_DIR ${Anasazi_SOURCE_DIR}/testmatrices
+    SOURCE_FILES bcsstk14.hb
+    EXEDEPS Tpetra_minres_hb
+  )
+
+  ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_Ifpack)
+  IF(${PACKAGE_NAME}_ENABLE_Ifpack)
```

Comment:
Remove the ifpack dependency here. The epetra Minres test required this, but yours here doesn't. Your test uses Belos-only constructs. [The true analogy to the ifpack dependency would be to use an ifpack2 preconditioner, but that would create a circular dependency between belos and ifpack2.  I like what you have done here to test the preconditioning.]

Reply:
Thanks for this comment, I will fix it and thank you, I understand better the dependencies between Belos and the other packages. [If we were to use Ifpack2 in Belos tests in the future, how would we go about it? Thanks in advance for your help!]